nautobot_design_builder/design_job.py

```diff
@@ -290,8 +303,12 @@ def _run_in_transaction(self, **kwargs):  # pylint: disable=too-many-branches, t
             job_result=self.job_result,
             extensions=extensions,
             change_set=change_set,
+            import_mode=data["import_mode"],
         )
 
+        if data["import_mode"] and data["deployment_name"]:
+            self.log_info(message=f'Running in import mode for {data["deployment_name"]}.')
+
```

Comment:
```suggestion
        if data["import_mode"]:
            self.log_info(message=f'Running in import mode for {data["deployment_name"]}.')
```

The second expression in the conditional is unneeded because import mode cannot be `True` if the design is not a deployment job.